### PR TITLE
fix zpool_iostat plugin

### PR DESCRIPTION
**Pull request by MartinNowak**
- use /bin/sh instead of /bin/bash
- add autoconf capabilities

### CODE REVIEW

File: plugins/zfs/zpool_iostat [hunk 1]
```diff
@@ -1,4 +1,7 @@
-#!/bin/bash
+#!/bin/sh
+
+#%# family=auto
+#%# capabilities=autoconf
```
Comment (ssm) [inline plugins/zfs/zpool_iostat]: To add "autoconf" to capabilities, the autoconf itself should be able to say both "yes" and "no", after checking for the presence of whatever's being monitored.  For now, it just says "yes", which would be wrong on any system not using zfs.

File: plugins/zfs/zpool_iostat [hunk 2]
```diff
@@ -1,4 +1,7 @@
-#!/bin/bash
+#!/bin/sh
```
Comment (ssm) [inline plugins/zfs/zpool_iostat]: :+1:

File: plugins/zfs/zpool_iostat (hunk 1 above)
Comment (ssm) [inline plugins/zfs/zpool_iostat]: I'll rewrite the required autoconf function, unless someone else beats me to it. :)

**Comment by MartinNowak** [comment]
Neither using zfs nor munin any more.

**Pull request closed without merging**